I don't understand why this is failing CI, the error reported seems to only show up in the Nix flake build (which uses Ninja), and reports 
```
yosys> /build/source/kernel/yosys_common.h:23:10: fatal error: 'array' file not found
```
But I can't reproduce this locally, or see a reason why the changes here would make a difference when trying to build Yosys (and even while testing as far as I understand Ninja builds should ignore the `JOB_SERVER_AWARE` line).

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -31,6 +31,10 @@ include(YosysAbcSubmodule)
 include(YosysVerific)
 include(UseHomebrew)
 
+# Fix for using Clang from nixpkgs
+# see https://github.com/YosysHQ/yosys/pull/5936#issuecomment-4637319568
+set(CMAKE_CXX_SCAN_FOR_MODULES NO)
+
 # Build options.
 set(YOSYS_COMPILER_LAUNCHER "" CACHE STRING "Compiler launcher (ccache, sccache)")
 option(YOSYS_ENABLE_COVERAGE "Enable code coverage" OFF)

